server/router/pom.xml

```diff
@@ -194,30 +194,12 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <configuration combine.self="override">
+                <configuration>
                     <trimStackTrace>false</trimStackTrace>
                     <parallel>suitesAndClasses</parallel>
```

Comment:
Did you forgot to change to 'none' here ?

Reply:
Actually, that module works with the parallel testsuite

Reply:
Nice! This is what I thought but looking to your comment above and modifications, I wanted to double check just in case! LGTM, but you asked @danberindei for a review so I'm not merging then